man/ocaml.1

```diff
@@ -37,23 +37,19 @@ command is the toplevel system for OCaml,
 that permits interactive use of the OCaml system through a
 read-eval-print loop. In this mode, the system repeatedly reads OCaml
 phrases from the input, then typechecks, compiles and evaluates
-them, then prints the inferred type and result value, if any. The
+them, then prints the inferred type and result value, if any.
 system prints a # (hash) prompt before reading each phrase.
 
-A toplevel phrase can span several lines. It is terminated by ;; (a
-double-semicolon). The syntax of toplevel phrases is as follows.
+Input to the toplevel can span several lines. It begins after the #
+(sharp) prompt printed by the system and is terminated by ;; (a
+double-semicolon) followed by optional white space and an end of line.
+The toplevel input consists in one or several toplevel phrases.
 
-The toplevel system is started by the command
-.BR ocaml (1).
-Phrases are read on standard input, results are printed on standard
-output, errors on standard error. End-of-file on standard input
-terminates
-.BR ocaml (1).
```

Comment:
Why is this removed, is it redundant with some other place?

Reply:
It's both wrong (everything is printed on stdout) and mostly redundant with the above, except for the bit about end-of-file. I've added it back.